src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -860,16 +861,120 @@ case class Check(
       includeUpperBound: Boolean = true,
       hint: Option[String] = None)
     : CheckWithLastConstraintFilterable = {
+      isContainedInRange(column, lowerBound, upperBound, includeLowerBound, includeUpperBound, hint)
+  }
 
+  /**
+    * Asserts that the non-null values in a column fall into the predefined interval
+    *
+    * @param column column to run the assertion
+    * @param lowerBound lower bound of the interval
+    * @param upperBound upper bound of the interval
+    * @param includeLowerBound is a value equal to the lower bound allows?
+    * @param includeUpperBound is a value equal to the upper bound allowed?
+    * @param hint A hint to provide additional context why a constraint could have failed
+    * @param quoteBounds should the bound values be quoted?
```

Comment:
when/why should they be quoted?

Reply:
The data types that need quoting are a superset of `{ DateType, TimestampType, StringType }`. These types need quoting when used in a query as below.

Reply:
Can you add this information to the comment?